modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/AttachmentObjectFieldBusinessType.java

```diff
@@ -246,7 +248,8 @@ private void _validateObjectFieldSettingFileSource(
 			String objectFieldName, String objectFieldSettingValue)
 		throws PortalException {
 
-		if (!Objects.equals(objectFieldSettingValue, "itemSelector") &&
+		if ((!GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-146523")) ||
```

Comment:
If the feature flag is `false`, shouldn't we keep verifying the file source `documentsAndMedia`?

Reply:
I was thinking that if the feature flag is false then the user should not be able to set this value until the functionality is fully implemented.

Reply:
You're right about that. My question was related to the old behavior that validated the `documentsAndMedia` setting's value.

Reply:
hmm, got it.. It was a mistake to add this validation in the previous story 😢 If I change back to `documentsAndMedia` I would have to keep the old behavior anyway, right?

Reply:
There won't be necessary anymore. 😄